wordpress-mobile/WordPress-iOS
# Add 'Sign in with Apple' entitlement.

Fixes #n/a

This adds the 'Sign in with Apple' entitlement. It's not used for anything yet, so testing is primarily to ensure the app still works.

To test:
- Open the app in Xcode 11.
  - Verify it runs with no problems.
  - Go to the project > WordPress target > Signing & Capabilities. 
  - Verify the 'Sign in with Apple' entitlement appears (probably at the bottom).
- Run the app in Xcode 10. Verify it runs with no problems.

Update release notes:
